File: src/main/java/com/sbna/fxfeed/model/MarketPrice.java
package com.sbna.fxfeed.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Getter
@Entity
@AllArgsConstructor
@NoArgsConstructor
public class MarketPrice {
    @Id
    Long id;

    @Column(nullable = false)
    String name;

    @Column(nullable = false)
    Double orgBidPrice;

    @Column(nullable = false)
    Double orgAskPrice;

    @Column(nullable = false)
    Double bidPrice;

    @Column(nullable = false)
    Double askPrice;

    @Column(nullable = false)
    LocalDateTime time;
}

File: src/main/kotlin/com/sbna/fxfeed/model/MarketPrice.kt
package com.sbna.fxfeed.model

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import java.time.LocalDateTime

@Entity
class MarketPrice (
    @Id
    val id: Long,
    @Column(nullable = false)
    val name : String,
    @Column(nullable = false)
    val orgBidPrice : Double,
    @Column(nullable = false)
    val orgAskPrice : Double,
    @Column(nullable = false)
    val bidPrice : Double,
    @Column(nullable = false)
    val askPrice : Double,
    @Column(nullable = false)
    val time : LocalDateTime
)